### DISCUSSION TITLE
Application Identifier was not found in the directory

### DISCUSSION THREAD

**Discussion by Koops1974** [Micke-K/IntuneManagement#230] [General]
Hi 

I have been using the app for a while with no errors.

But now I am getting **"Application Identifier was not found in the directory"**

From what I can work out this need to be added into the **Entra/Applicaitons/App Registrations/** and then it should run.

Could someone confirm what I need to do as I have tried a few times but its still failing.

Thanks for any help given.

**Comment by Micke-K** [comment]
Hello,

Which app are you using? The app for the Intune PowerShell module has been decommissioned by Microsoft. If that is the case, then you gave to change the app in Settings and use the Graph API app instead. 

If you want to use a custom app, you can follow the instructions in [Issue 229](https://github.com/Micke-K/IntuneManagement/issues/229). That also has a link to Microsoft's documentation on how to create an app. 

Let me know how it goes. 

Cheers!

**Comment by Koops1974** [comment]
I downloaded the app again and it worked.
When it loaded I had the change to Graph API option.

Thanks for you help......this is a great application.
